lib/web_ui/lib/src/engine/canvaskit/text.dart

```diff
@@ -843,7 +844,7 @@ class CkParagraphBuilder implements ui.ParagraphBuilder {
         _styleStack = <CkTextStyle>[],
         _paragraphBuilder = canvasKit.ParagraphBuilder.MakeFromFontProvider(
           style.skParagraphStyle,
-          skiaFontCollection.fontProvider,
+          (renderer.fontCollection as SkiaFontCollection).fontProvider,
```

Comment:
Can probably use `canvasKitRenderer` instead to avoid the `as SkiaFontCollection`?
```suggestion
          canvasKitRenderer.fontCollection.fontProvider,
```